org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -1343,13 +1343,44 @@ protected void load (int ID, String trxName)
 		}
 	}	//	load
 
+	/**
+	 * Load record with UUID
+	 * 
+	 * @param uuID    UUID
+	 * @param trxName transaction name
+	 */
+	public void loadByUU(String uuID, String trxName)
+	{
+		// reset new values
+		m_newValues = new Object[get_ColumnCount()];
+		checkImmutable();
+
+		if (log.isLoggable(Level.FINEST))
+			log.finest("uuID=" + uuID);
+		if (!Util.isEmpty(uuID, true))
+		{
+			load(uuID,trxName);
+		}else {
+			throw new IllegalArgumentException("Invalid null UU - Must pass valid UU");
+		}
+	} // loadByUU
 
 	/**
 	 *  (re)Load record with m_ID[*]
 	 *  @param trxName transaction
 	 *  @return true if loaded
 	 */
-	public boolean load (String trxName)
+	public boolean load (String trxName) {
+		return load(null, trxName);
+	}
+	
+	/**
+	 *  (re)Load record with m_ID[*]
```

Comment:
the javadoc should be updated to reflect the addition of uuID option.

Reply:
Fixed